src/v/cluster/partition_probe.cc

```diff
@@ -48,9 +48,10 @@ void replicated_partition_probe::setup_internal_metrics(const model::ntp& ntp) {
     auto ns_label = sm::label("namespace");
     auto topic_label = sm::label("topic");
     auto partition_label = sm::label("partition");
-    auto aggregate_labels = config::shard_local_cfg().aggregate_metrics()
-                              ? std::vector<sm::label>{sm::shard_label}
-                              : std::vector<sm::label>{};
+    auto aggregate_labels
+      = config::shard_local_cfg().aggregate_metrics()
+          ? std::vector<sm::label>{sm::shard_label, partition_label}
+          : std::vector<sm::label>{};
```

Comment:
What does it mean to be `is_elected_leader` of a topic?

Reply:
>What does it mean to be is_elected_leader of an aggregation of partitions?
>Same for offsets.

i guess it would sort of be like `leader count`? but for offsets, yeh, that doesn't seem to aggregate in a way that provides meaning. wdyt @ballard26

Reply:
I think I was probably a bit over-zealous in my pursuit of uniformity. For offsets you could interpret it as rate per-shard rate of offset changes. It definitely does reduce the usefulness of the metric though. I'll be closing this PR and opening a new one with a much reduced set of new aggregations.